docs/sequence/PISPLinkingOTPApi.puml

```diff
@@ -237,11 +244,11 @@ FSIOP-Destination: pispa
 end note
 Switch --> DFSP_LM: 202 ACCEPTED
 deactivate DFSP_LM
-activate PISP_TP_IN
 Switch ->  PISP_TP_IN: POST /consents
+activate PISP_TP_IN
 PISP_TP_IN -> PISP_TP_IN: const model = await loadFromKVS({key: 6789})
-activate PISP_LM
 PISP_TP_IN -> PISP_LM: model.registerCredential()
```

Comment:
You're correct. Oh boy...this is quite a big oversight on my end. This might require a new endpoint such as /linking/request-consent/6789/sendCredential so the PISP can pass in the credential. Will need to visit error scenarios as well maybe.

Can I leave this for a follow up PR since it is quite a big change to the happy path? Where this PR is more focusing on error scenarios.

Reply:
Sure, sounds good to me.